### ISSUE TITLE
NavigationCardStackPanResponder doesn't work.

### ISSUE THREAD

**Issue by keeleycarrigan** [facebook/react-native#11393]
### Description

When I use `NavigationCardStackPanResponder` as the pan responder in `NavigationTransitioner` there is no response to swiping back. `NavigationPagerPanResponder` does work even when using the card stack interpolator, but the animation stutters a little in iOS and a lot on Android regardless of which interpolator I use. If I'm missing something please let me know. I've only been messing around with NavigationExperimental for a few days.

### Reproduction

https://github.com/keeleycarrigan/test-rn-navigation-experimental

You should be able to test out the different responders and interpolator styles by changing `navStyle` and `navPanResponder` attributes [here](https://github.com/keeleycarrigan/test-rn-navigation-experimental/blob/master/src/components/home.js#L67). Please forgive the messy project. I'm just messing around with NavigationExperimental for the first time so I haven't paid much attention to organization :).

### Solution

Wish I had one.

### Additional Information

* React Native version: 0.38
* Platform: both
* Operating System: MacOS

**Comment by ericvicenti** [comment]
Unfortunately I don't have much to go on. I looked at your code but couldn't find anything obviously wrong with it. Can you reproduce the issue in a simpler way?

I wouldn't say that the pan responder "doesn't work", because it seems to work fine in NavigationCardStack. Although I agree there isn't much of an API for re-configuring pan responders, you should be able to copy the whole NavigationCardStack and just change the responder.

**Comment by keeleycarrigan** [comment]
I guess what I mean by "doesn't work" is when I swipe there's no recognition that I've swiped when I used the card stack responder. If I change the responder to the pager responder no matter what interpolator or method of animation I use that will work. Since I posted this I've implemented NavigationExperimental in a private app I'm developing and get the same results. [This repo](https://github.com/tkjone/react-native-examples), although it has some issues, also tries to implement the card stack responder. After fixing some errors that prevent the 'back swipe', only when the responder is changed to the pager responder does it work.

Sorry I can't give more info right now. I have been digging into the NavigationExperimental source a little more lately so I'll try to see if I can debug a little better now that I'm more familiar with the source code. Thanks!
